addon/components/sortable-group.js

```diff
@@ -111,7 +118,11 @@ export default Component.extend({
         items.invoke('thaw');
       });
     });
-
-    this.sendAction('onChange', models);
+    
+    if (groupModel === null) {
```

Comment:
Ah, missed the const definition when reading over it, so I was scratching my head on the definition of `NO_MODEL`, and assumed that it's a null equivalent. That should do the trick. Let me update it.